docs/swarm/swarm-mode.md

```diff
@@ -73,10 +73,31 @@ To add a manager to this swarm, run the following command:
     192.168.99.100:2377
 ```
 
+### Configure the advertise address
+
+Manager nodes use an advertise address to allow other nodes in the swarm access
+to the API and overlay networking. The other nodes on the swarm must be able to
+access the manager node on its advertise address IP address.
+
+If you don't specify an advertise address, Docker checks if the system has a
+single IP address. If so, Docker uses the IP address with with the listening
+port `2377` by default. If the system has multiple IP addresses, you must
+specify the correct  `--advertise-addr` to enable inter-manager communication
+and overlay networking:
```

Comment:
I wrongly put this comment in the "Joining" document but it actually belongs here:

Another thing we should mention here is that when you set up the swarm with swarm init, you need to specify the --advertise-addr if the address where other nodes will use to reach the first node is not the same as the address it sees as its own. One example of where this comes into play is a cloud setup that spans different regions (or availability zones?). In a lot of cloud setups, hosts have internal addresses, which are the ones you see with ifconfig, and external addresses that you use from outside that zone or region. The first manager needs to be told about its external address with --advertise-addr so that it can spread that information to other nodes that will connect to it in the future.